<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ome_maps_viewer_dialog</class>
 <widget class="QDialog" name="ome_maps_viewer_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>728</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QGroupBox" name="seed_search_group_box">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Seed Search</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="omega_group_box">
        <property name="title">
         <string>Omega</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="tolerance_label">
           <property name="text">
            <string>Tolerance:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ScientificDoubleSpinBox" name="omega_tolerance">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ScientificDoubleSpinBox" name="fiber_step">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="method_group_box">
        <property name="title">
         <string>Method</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="1">
          <widget class="QTabWidget" name="tab_widget">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="label_tab">
            <attribute name="title">
             <string>Label</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="QLabel" name="label_threshold_label">
               <property name="text">
                <string>Threshold:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="filter_radius_label">
               <property name="text">
                <string>Filter Radius: </string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1">
              <widget class="ScientificDoubleSpinBox" name="filter_radius">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="maximum">
                <double>10000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="ScientificDoubleSpinBox" name="label_threshold">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="blob_dog_tab">
            <attribute name="title">
             <string>Blob Dog</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="1">
              <widget class="ScientificDoubleSpinBox" name="bd_min_sigma">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="bd_sigma_ratio_label">
               <property name="text">
                <string>Sigma Ratio:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="bd_max_sigma_label">
               <property name="text">
                <string>Max Sigma:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="ScientificDoubleSpinBox" name="bd_max_sigma">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>5.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="bd_min_sigma_label">
               <property name="text">
                <string>Min Sigma:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="ScientificDoubleSpinBox" name="bd_sigma_ratio">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.600000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="bd_threshold_label">
               <property name="text">
                <string>Threshold:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="ScientificDoubleSpinBox" name="bd_threshold">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="bd_overlap_label">
               <property name="text">
                <string>Overlap:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="ScientificDoubleSpinBox" name="bd_overlap">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="value">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="blob_log_tab">
            <attribute name="title">
             <string>Blob Log</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="3" column="0">
              <widget class="QLabel" name="bl_sum_sigma_label">
               <property name="text">
                <string>Num Sigma:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="bl_overlap_label">
               <property name="text">
                <string>Overlap:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="ScientificDoubleSpinBox" name="bl_max_sigma">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>5.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="bl_max_sigma_label">
               <property name="text">
                <string>Max Sigma:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="bl_threshold_label">
               <property name="text">
                <string>Threshold:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="bl_min_sigma_label">
               <property name="text">
                <string>Min Sigma:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="ScientificDoubleSpinBox" name="bl_min_sigma">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="bl_num_sigma">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="ScientificDoubleSpinBox" name="bl_threshold">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="ScientificDoubleSpinBox" name="bl_overlap">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>8</number>
               </property>
               <property name="minimum">
                <double>-1000000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1000000.000000000000000</double>
               </property>
               <property name="value">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="method">
           <item>
            <property name="text">
             <string>Label</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blob Dog</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blob Log</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="eta_group_box">
        <property name="title">
         <string>Eta</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="eta_tolerance_label">
           <property name="text">
            <string>Tolerance:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ScientificDoubleSpinBox" name="eta_tolerance">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="eta_mask_label">
           <property name="text">
            <string>Mask:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ScientificDoubleSpinBox" name="eta_mask">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fiber_step_label">
        <property name="text">
         <string>Fiber Step:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QGroupBox" name="clustering_group_box">
        <property name="title">
         <string>Clustering</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="1" column="0">
          <widget class="QLabel" name="clustering_completeness_label">
           <property name="text">
            <string>Completeness:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="clustering_algorithm_label">
           <property name="text">
            <string>Algorithm:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="clustering_radius_label">
           <property name="text">
            <string>Radius:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ScientificDoubleSpinBox" name="clustering_radius">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="clustering_algorithm">
           <item>
            <property name="text">
             <string>DBScan</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SPH-DBScan</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ORT-DBScan</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FClusterData</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ScientificDoubleSpinBox" name="clustering_completeness">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>0.850000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <layout class="QVBoxLayout" name="canvas_layout"/>
   </item>
   <item row="0" column="2" rowspan="2">
    <layout class="QGridLayout" name="grid_layout">
     <item row="2" column="2">
      <widget class="QCheckBox" name="label_spots">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Label Spots</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="export_button">
       <property name="text">
        <string>Export</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="4">
      <layout class="QVBoxLayout" name="color_map_editor_layout"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="active_hkl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="active_hkl_label">
       <property name="text">
        <string>Displayed hkl</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="4">
      <widget class="QGroupBox" name="select_hkls_group">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>HKL Seeds</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="select_hkls_widget_layout"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>method</tabstop>
  <tabstop>tab_widget</tabstop>
  <tabstop>filter_radius</tabstop>
  <tabstop>label_threshold</tabstop>
  <tabstop>bd_min_sigma</tabstop>
  <tabstop>bd_max_sigma</tabstop>
  <tabstop>bd_sigma_ratio</tabstop>
  <tabstop>bd_threshold</tabstop>
  <tabstop>bd_overlap</tabstop>
  <tabstop>bl_min_sigma</tabstop>
  <tabstop>bl_max_sigma</tabstop>
  <tabstop>bl_num_sigma</tabstop>
  <tabstop>bl_threshold</tabstop>
  <tabstop>bl_overlap</tabstop>
  <tabstop>fiber_step</tabstop>
  <tabstop>omega_tolerance</tabstop>
  <tabstop>eta_tolerance</tabstop>
  <tabstop>eta_mask</tabstop>
  <tabstop>clustering_radius</tabstop>
  <tabstop>clustering_completeness</tabstop>
  <tabstop>clustering_algorithm</tabstop>
  <tabstop>active_hkl</tabstop>
  <tabstop>label_spots</tabstop>
  <tabstop>export_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>ome_maps_viewer_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>687</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>ome_maps_viewer_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>687</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
